java/custom-resource/src/main/java/software/amazon/awscdk/examples/CustomResourceApp.java

```diff
@@ -0,0 +1,13 @@
+package software.amazon.awscdk.examples;
+
+import software.amazon.awscdk.core.App;
+
+public class CustomResourceApp {
+    public static void main(final String args[]) {
+        App app = new App();
+        
+        new CustomResourceStack(app, "cdk-custom-resource-example2");
+
+        app.synth();
```

Comment:
app.synth() is no longer required, this can be removed from all 3 examples.

Reply:
Wait, I lied about this. We are recommending to leave this in for all languages except typescript for now so you're right on this.